apps/src/templates/projects/FeaturedProjectsTable.jsx

```diff
@@ -168,6 +168,25 @@ const actionsFormatterUnfeatured = (actions, {rowData}) => {
   );
 };
 
+const actionsFormatterSaved = (actions, {rowData}) => {
+  return (
+    <QuickActionsCell>
+      <PopUpMenu.Item
+        onClick={() => feature(rowData.channel, rowData.publishedAt)}
+      >
+        Feature
```

Comment:
This is confusing to me - are you featuring or saving here?

Reply:
Featuring. Saving is done from the gray admin UI box on the individual project page.